ansible/roles/setup-kafka/tasks/main.yml

```diff
@@ -12,3 +12,11 @@
   when: kafka_id=="1" and item.retention_time is defined  
   tags:
     - processing-kafka
+
+- name: create topics
+  command: /opt/kafka/bin/kafka-topics.sh --zookeeper localhost:2181 --create --topic {{env_name}}.{{item.name}} --partitions {{ item.num_of_partitions }} --replication-factor {{ item.replication_factor }}
+  with_items: "{{ml_service_topics}}"
+  ignore_errors: true
+  when: kafka_id=="1"
+  tags:
+    - ml-service-topics
```

Comment:
keep the tag name as it is)processing-kafka)  and update the when condition (check if ml-servive-deploy is set to true)

Reply:
changed the tag name

Reply:
when condtion unable to keep like this
when: kafka_id=="1" and ml_service_topics.deploy == true
 when: kafka_id=="1" and ml_service_topics is defined
so i have defined like this is ok @G33tha